pallets/handles/src/handles-utils/constants.rs

```diff
@@ -2,27 +2,88 @@
-	0x0500..=0x052F, // Cyrillic Supplementary
-	0x0590..=0x05FF, // Hebrew
-	0x0600..=0x06FF, // Arabic
-	0x0900..=0x097F, // Devanagari
-	0x0980..=0x09FF, // Bengali
-	0x0E00..=0x0E7F, // Thai
-	0x1100..=0x11FF, // Hangul Jamo
-	0x1E00..=0x1EFF, // Latin Extended Additional
-	0x1F00..=0x1FFF, // Greek Extended
-	0x3040..=0x309F, // Hiragana
-	0x30A0..=0x30FF, // Katakana
-	0x3400..=0x4DBF, // CJK Unified Ideographs Extension A
-	0x4E00..=0x9FFF, // CJK Unified Ideographs
-	0xAC00..=0xD7AF, // Hangul Syllables
-	0xF900..=0xFAFF, // CJK Compatibility Ideographs
-	0xFB50..=0xFDFF, // Arabic Presentation Forms-A
+/// Characters that are allowed.
+#[rustfmt::skip]
+pub const ALLOWED_UNICODE_CHARACTER_RANGES: [RangeInclusive<u16>; 81] = [
+    0x0020..=0x007A, // BasicLatin
+    0x0080..=0x00FF, // Latin-1 Supplement
+    0x0100..=0x017F, // Latin Extended-A
+    0x180..=0x24F,   // Latin Extended-B
+    0x02B0..=0x02FF, // Spacing Modifier Letters
+    0x0300..=0x036F, // Combining diacritical marks
+    0x0370..=0x03FF, // Greek and Coptic
+    0x0400..=0x04FF, // Cyrillic
+    0x0531..=0x058A, // Armenian
+    0x0591..=0x05F4, // Hebrew
+    0x0600..=0x06FF, // Arabic
+    0x0700..=0x074F, // Syriac
+    0x0750..=0x077F, // ArabicSupplement
+    0x0780..=0x07B1, // Thaana
+    0x07C0..=0x07FA, // NKo
+    0x0800..=0x083E, // Samaritan
+    0x0840..=0x085E, // Mandaic
+    0x0900..=0x097F, // Devanagari
+    0x0981..=0x09FB, // Bengali
+    0x0A01..=0x0A75, // Gurmukhi
+    0x0A81..=0x0AF1, // Gujarati
+    0x0B01..=0x0B77, // Oriya
+    0x0B82..=0x0BFA, // Tamil
+    0x0C01..=0x0C7F, // Telugu
+    0x0C82..=0x0CF2, // Kannada
+    0x0D02..=0x0D7F, // Malayalam
+    0x0D82..=0x0DF4, // Sinhala
+    0x0E01..=0x0E5B, // Thai
+    0x0E81..=0x0EDD, // Lao
+    0x0F00..=0x0FDA, // Tibetan
+    0x1000..=0x109F, // Myanmar
+    0x10A0..=0x10FC, // Georgian
+    0x1100..=0x11FF, // HangulJamo
+    0x1200..=0x137C, // Ethiopic
+    0x1380..=0x1399, // EthiopicSupplement
+    0x13A0..=0x13F4, // Cherokee
+    0x1400..=0x167F, // UnifiedCanadianAboriginalSyllabics
+    0x1680..=0x169C, // Ogham
+    0x16A0..=0x16F0, // Runic
+    0x1700..=0x1714, // Tagalog
+    0x1720..=0x1736, // Hanunoo
+    0x1740..=0x1753, // Buhid
+    0x1760..=0x1773, // Tagbanwa
+    0x1780..=0x17F9, // Khmer
+    0x1800..=0x18AA, // Mongolian
+    0x18B0..=0x18F5, // Unified Canadian Aboriginal Syllabics Extended
+    0x1900..=0x194F, // Limbu
+    0x1950..=0x1974, // TaiLe
+    0x1980..=0x19DF, // NewTaiLue
+    0x19E0..=0x19FF, // KhmerSymbols
+    0x1A00..=0x1A1F, // Buginese
+    0x1A20..=0x1AAD, // TaiTham
+    0x1B00..=0x1B7C, // Balinese
+    0x1B80..=0x1BB9, // Sundanese
+    0x1BC0..=0x1BFF, // Batak
+    0x1C00..=0x1C4F, // Lepcha
+    0x1C50..=0x1C7F, // Ol Chiki
+    0x1E00..=0x1EFF, // Latin Extended Additional
+    0x1F00..=0x1FFF, // Greek Extended
+    0x200C..=0x206F, // General punctuation, used in some languages to indicate syllables such as glottal stops
```

Comment:
Not including this was causing some languages to fail even though they used things like Latin Extended-A otherwise